code/modules/paperwork/paper.dm

```diff
@@ -106,6 +106,35 @@
 	add_fingerprint(usr)
 	return
 
+/obj/item/paper/verb/resize()
+	set name = "Resize paper"
+	set category = "Object"
+	set src in usr
+
+	var new_width = 400
+	var new_height = 400
+
+	new_width = input("Paper width", 400, new_width) as num|null
+	new_height = input("Paper height", 400, new_height) as num|null
+
+	new_width = limit_size(new_width)
+	new_height = limit_size(new_height)
+
+	paper_width = new_width
+	paper_height = new_height
+
+	return
+
+/obj/item/paper/proc/limit_size(num)
```

Comment:
I think `Clamp()` already does this?

Reply:
I had no idea how to track down what I needed thanks, I'll go ahead and make the change.